requirements.txt

```diff
@@ -1,4 +1,3 @@
--e git+https://github.com/oyvindrobertsen/abakaffe-cli.git@1193ecf05b311133e8347431e4932ba7c7c83255#egg=abakaffe_cli-refactor
```

Comment:
Det er du som har lagt den til for lenge siden. Den la en kopi av koden i `src/` som var fra refactor-branchen eller noe, og pip laget noen klage-filer. Men tror det skal funke fint uten.